lib/api/controllers/security.js

```diff
@@ -1402,12 +1255,12 @@ class SecurityController extends NativeController {
   }
 
   /**
- *
- * @param {Kuzzle} kuzzle
- * @param {string.<profile|role|user>} type
- * @param {Request} request
- * @returns {Promise<*>}
- */
+   *
+   * @param {Kuzzle} kuzzle
```

Comment:
I know you didn't change this method but can you remove the `kuzzle` parameter and use the `this.kuzzle` member instead please?